### DISCUSSION TITLE
Updating to @open-wc/scoped-elements v3

### DISCUSSION THREAD

**Discussion by lazka** [ing-bank/lion#2134] [General]
Just wondering if this project will be updating to @open-wc/scoped-elements v3? One change in v3 is that the polyfill is no longer optional and would be a hard dependency.

**Comment by gerjanvangeest** [comment]
@lazka We do not have concrete plans yet, but we are having some internal discussions about it. We'll keep you update if we have more info about it

Probably will happen at the same time as the update to Lit v3: https://github.com/ing-bank/lion/discussions/2111

**Reply by lazka** [reply]
Thanks, appreciated

**Reply by riovir** [reply]
I was worried that, running tests in JSDOM would be a problem, but it seems that they are working in the latest version of the polyfill + JSDOM combo.

**Reply by lazka** [reply]
I see it happened with https://github.com/ing-bank/lion/pull/2168 with some adjustments to make the polyfill optional again. great :)
